src/test/java/RunCucumberTest.java

```diff
@@ -8,9 +8,13 @@
 import java.util.Properties;
 
 @RunWith(Cucumber.class)
-@CucumberOptions(plugin = {"pretty", "html:target/cucumber-reports" },
+@CucumberOptions
+//        (plugin = {"json:target/cucumber.json"},
```

Comment:
Remove code if it is not required

Reply:
This code sets type of report, when we will start tests I'll change one option to other. I don't see reason to remove this row

Reply:
Make this configurable and move this to application.properties